src/ZkFold/Cardano/Scripts/SymbolicVerifier.hs

```diff
@@ -28,3 +29,13 @@ symbolicVerifier contract proof ctx =
 
         -- Computing public input from the transaction data
         input = toInput . dataToBlake $ (ins, refs, outs, range)
+
+untypedSymbolicVerifier :: SetupBytes -> BuiltinData -> BuiltinData -> BuiltinUnit
```

Comment:
Suggest adding INLINABLE pragma.  Also, type signature should be `SetupBytes -> BuiltinData -> BuiltinUnit`.  See suggestions for file `src/ZkFold/Cardano/Scripts/PlonkVerifier.hs` .